packages/components/src/custom-select-control-v2/legacy-component/test/index.tsx

```diff
@@ -388,8 +388,9 @@ describe.each( [
 			await sleep();
 			await press.Tab();
 			expect( currentSelectedItem ).toHaveFocus();
+			expect( currentSelectedItem ).toHaveTextContent( 'violets' );
 
-			await type( 'aq' );
```

Comment:
Apparently the multi-character typeahead is causing the flakiness. I think it'd be an acceptable compromise to test only a single-character typeahead.

Reply:
Is it worth adding an inline comment here, and in the non-legacy component test, flagging this as deliberately avoiding the multi-character typeahead?

Reply:
Good idea, will do 👍